### ISSUE TITLE
Add results and severity in checks list readme

### ISSUE THREAD

**Issue by davidperezgar** [WordPress/plugin-check#758]
I see that we have the grouped checks in the documentation. I think It should be a good idea to add the results for every check and their seveiry levels in [checks lists documentation](https://github.com/WordPress/plugin-check/blob/trunk/docs/checks.md).

That gives us the status of all results that the plugin can make after a review.

**Comment by ernilambar** [comment]
Results? Can you please elaborate or provide example? What advantages do you see for listing such?

**Comment by davidperezgar** [comment]
Yes! For example the check obfuscated_code_detected, has these results:
- Source Guardian - Severity 7
- IonCube - Severity 7
- and more...
